app/src/main/java/org/wikipedia/analytics/eventplatform/MachineGeneratedArticleDescriptionsAnalyticsHelper.kt

```diff
@@ -27,15 +27,22 @@ class MachineGeneratedArticleDescriptionsAnalyticsHelper {
     }
 
     fun logAttempt(context: Context, finalDescription: String, wasChosen: Boolean, wasModified: Boolean, title: PageTitle) {
-        log(context, composeLogString(title) + ".attempt:$finalDescription" +
-                ".suggestion1:${encode(apiOrderList.first())}" + (if (apiOrderList.size > 1) ".suggestion2:${encode(apiOrderList.last())}" else "") +
-                getOrderString(wasChosen, chosenSuggestion) + ".modified:$wasModified.timeSpentMs:${timer.elapsedMillis}")
+        log(context, composeLogString(title) + ".attempt:$finalDescription ${getSuggestionOrderString(wasChosen, wasModified)}" +
```

Comment:
This leaves an unnecessary space between `$finalDescription` and `${getSuggestionOrderString`.